api/src/Modules/staff/ModuleStaff.php

```diff
@@ -40,5 +42,76 @@ public function handle(Request $request, Response $response, $data, $code)
     }
 
 
+    public function customizeAnnouncementRBAC($instance, $database)
+    {
+        $role = $instance->getRole('all.staff');
+        $role->addPermission('api.get.staff');
+
+        if (array_key_exists('IS_ADMIN', $_SESSION) && $_SESSION['IS_ADMIN']) {
```

Comment:
I am not sure if we should be checking IS_ADMIN here... IS_ADMIN is going to basically bypass all this logic anyway.  so we should not add anything to an admin role. 

Also you do not use `getRole` anymore

Reply:
I'm not so sure about that - without this addition in `ModuleStaff`, the admin user was not getting those permissions back when calling the API for member permissions, so their super-user abilities on this page were not rendered properly even if the API would have allowed them to make the request.

Reply:
With the current code if you register the permission then the admin use will have them in their list.